### ISSUE TITLE
No tray icon since 20.2.0 on Ubuntu 17.10 on snap

### ISSUE THREAD

**Issue by dimitarnestorov** [electron-userland/electron-builder#2776]
* **Version**: 20.2.0 and above

* **Target**: Linux Snap

I believe the main reason is `desktop-gdk3`. Is there a way to target the snap template to be version 1, cause everything works fine with it?

To make the tray icon show up I need to use either of those configurations:
- `electron@1.8.2` and `electron-builder@latest`
- `electron@latest` and `electron-builder@20.1.1`

**Comment by develar** [comment]
Hmm... we use desktop-gtk3 only for electron 2.

Please provide your snap configuration — I want to understand is template app is used or not.

**Comment by dimitarnestorov** [comment]
Configuration:
```
  "build": {
    "files": [
      "build",
      "index.js",
      "src",
      "icons",
      "renderer.js"
    ],
    "snap": {
      "category": "Utility"
    },
    "appId": "pm.ora",
    "copyright": "Copyright © 2018 Ora PM Ltd.",
    "productName": "Ora",
    "buildVersion": "19.0.6",
    "protocols": {
      "name": "Ora",
      "schemes": [
        "ora"
      ]
    },
    "linux": {
      "target": [
        "snap"
      ],
      "executableName": "ora",
      "icon": "icons/png/1024x1024.png"
    },
    "win": {
      "icon": "icons/win/icon.ico",
      "target": [
        {
          "target": "nsis",
          "arch": "x64"
        },
        {
          "target": "nsis",
          "arch": "ia32"
        },
        {
          "target": "appx",
          "arch": "x64"
        }
      ]
    },
    "mac": {
      "icon": "icons/mac/icon.icns",
      "target": [
        "mas-dev",
        "mas"
      ]
    },
    "mas": {
      "entitlements": "./config/entitlements/parent.plist",
      "entitlementsInherit": "./config/entitlements/child.plist"
    },
    "appx": {
      "identityName": "Codemotion.Ora-TaskManagement",
      "applicationId": "Ora",
      "displayName": "Ora",
      "publisher": "CN=A6A54C5B-0F15-474E-B6FA-C5E1B5067B57",
      "publisherDisplayName": "Codemotion",
      "backgroundColor": "transparent"
    }
  }
```

-------

> we use desktop-gtk3 only for electron 2.

I disagree: https://github.com/electron-userland/electron-builder/blob/master/packages/electron-builder-lib/src/linuxPackager.ts#L26

**Comment by dimitarnestorov** [comment]
Fixed in 726706c43b55d5b38e6bb59891e52f96fd53197c (v20.11.0)

Thanks!
